pkg/kubectl/cmd/util/factory_object_mapping.go

```diff
@@ -330,19 +330,21 @@ func (f *ring1Factory) Reaper(mapping *meta.RESTMapping) (kubectl.Reaper, error)
 func (f *ring1Factory) HistoryViewer(mapping *meta.RESTMapping) (kubectl.HistoryViewer, error) {
 	mappingVersion := mapping.GroupVersionKind.GroupVersion()
 	clientset, err := f.clientAccessFactory.ClientSetForVersion(&mappingVersion)
+	external, err := f.clientAccessFactory.KubernetesClientSet()
```

Comment:
naming nit: if we need both internal and external clientset, name the internal one "internal" or name the external one "externalClientset"